docs/usage-with-routing.stories.mdx

```diff
@@ -44,6 +45,18 @@ const CarbonLinkWithReactRouter = ({ to, children }) => {
   return <Link onClick={onClick}>{children}</Link>;
 };
 
+const CarbonTabsWithReactRouter = ({ children }) => {
+  const history = useHistory();
+  const handleTabChange = useCallback(
+    (tabId) => {
+      history.push(tabId);
+    },
+    [history]
+  );
+
+  return <Tabs onTabChange={handleTabChange}>{children}</Tabs>;
```

Comment:
We need a `selectedTabId` on this as well don't we?

Reply:
Yep, I'm on fire 🔥  with this Pr